### DISCUSSION TITLE
Should we add union list species to GRIIS when they are absent ?

### DISCUSSION THREAD

**Discussion by SanderDevisscher** [inbo/alien-species-portal#10] [Q&A]
@timadriaens denk jij dat het nuttig/nodig is om soorten die wel op unie lijst staan en niet op de GRIIS - checklist toe te voegen aan de soortenlijst ?

**Comment by timadriaens** [comment]
Ik denk dat je er alleen informatie (indicatoren, kaarten, pathways etc.) over kan opvragen als ze effectief voorkomen (of voorkwamen) maw als er gbif records van zijn/waren. Dus ik zou denken van niet. Van zodra er eentje opduikt zal die dan wel erop komen hé... @damianooldoni is de Unielijst al gepubliceerd op gbif ondertussen of gebruiken we nog altijd die versie op git?

**Reply by damianooldoni** [reply]
Nee, de Unielijst is niet gepubliceerd op GBIF want we de infos via EASIN moeten opvragen (https://github.com/riparias/union-list/issues/6) en geen tijd gehad om verder eraan te werken.
